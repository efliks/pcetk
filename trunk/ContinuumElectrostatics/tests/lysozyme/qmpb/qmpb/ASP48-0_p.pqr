../0/site_ASP_48_p.pqr