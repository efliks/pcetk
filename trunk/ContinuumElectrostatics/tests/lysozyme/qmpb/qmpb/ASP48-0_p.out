Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named ASP48-0_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (15.754, 22.874, 30.005) 101 1
Finer level:    (15.754, 22.874, 30.005) 101 0.5
Finest level:   (15.754, 22.874, 30.005) 101 0.25


Self energy of ASP48-0_p in three dielectric environment = 184.74
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of ASP48-0_p with background charge set in three dielectric environment = -0.465169
(probably in kcal/mole)



Interaction energies of ASP48-0_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.000269547
0   1 : -3.43553e-06
1   0 : -1.05649e-05
1   1 : 0.000277236
2   0 : -0.00016559
2   1 : -3.57977e-06
3   0 : -0.000740729
3   1 : -8.35752e-05
3   2 : 4.52533e-05
3   3 : 0.000645467
4   0 : -8.41374e-05
4   1 : 0.000352284
5   0 : -0.000226167
5   1 : 0.00142729
6   0 : 0.00027682
6   1 : 0.00251058
7   0 : -0.000878011
7   1 : -2.93468e-05
8   0 : -0.000426993
8   1 : 0.0211451
9   0 : 369.481
9   1 : 484.962
10   0 : 0.00505114
10   1 : 0.0642412
11   0 : -0.00475238
11   1 : 0.0173801
12   0 : 0.00285341
12   1 : 0.03022
13   0 : -2.83364e-05
13   1 : 0.000460849
14   0 : -0.000974568
14   1 : -4.72789e-05
15   0 : -0.00170176
15   1 : -4.85737e-06
16   0 : -0.000390696
16   1 : 0.00892108
17   0 : -0.00227222
17   1 : 2.28678e-05
18   0 : 2.59066e-06
18   1 : 0.000270674
Total runtime: 7.32 s user 0.06 s system 
